extensions/npm/src/npmView.ts

```diff
@@ -276,7 +279,15 @@ export class NpmScriptsTreeDataProvider implements TreeDataProvider<TreeItem> {
 		return fullName === task.name;
 	}
 
-	private buildTaskTree(tasks: TaskWithLocation[]): Folder[] | PackageJSON[] | NoScripts[] {
+	private sortTaskTree(taskTree: TaskTree) {
+		return taskTree.sort((first, second) => {
+			const firstLabel = first.label?.toString() || '';
```

Comment:
Instead of doing `toString()`, it would be better to check if the `label` is a string or a `TreeItemLabel`, and if it's the later, to grab its `.label` property.

Reply:
@alexr00 done! Please take a look if it's good now.